cranelift/codegen/src/binemit/mod.rs

```diff
@@ -74,11 +81,11 @@ impl fmt::Display for Reloc {
             Self::X86CallPLTRel4 => write!(f, "CallPLTRel4"),
             Self::X86GOTPCRel4 => write!(f, "GOTPCRel4"),
             Self::Arm32Call | Self::Arm64Call => write!(f, "Call"),
-
             Self::ElfX86_64TlsGd => write!(f, "ElfX86_64TlsGd"),
             Self::MachOX86_64Tlv => write!(f, "MachOX86_64Tlv"),
             Self::Aarch64TlsGdAdrPage21 => write!(f, "Aarch64TlsGdAdrPage21"),
             Self::Aarch64TlsGdAddLo12Nc => write!(f, "Aarch64TlsGdAddLo12Nc"),
+            Self::RiscvCall => write!(f, "riscvCall"),
```

Comment:
Can you move this to just below `Self::Arm32Call | Self::Arm64Call => write!(f, "Call")` and keep the newline below that? The newline separates all TLS relocations from the rest. Also please capitalize as `RiscvCall` for consistency.